projects/OG-Analytics/src/main/java/com/opengamma/analytics/financial/interestrate/future/derivative/InterestRateFutureOptionPremiumSecurity.java

```diff
@@ -5,42 +5,13 @@
  */
 package com.opengamma.analytics.financial.interestrate.future.derivative;
 
-import org.apache.commons.lang.ObjectUtils;
-
-import com.opengamma.analytics.financial.interestrate.InstrumentDerivative;
 import com.opengamma.analytics.financial.interestrate.InstrumentDerivativeVisitor;
 import com.opengamma.util.ArgumentChecker;
-import com.opengamma.util.money.Currency;
 
 /**
  * Description of an interest rate future option with up-front margin security.
```

Comment:
change description to "Description of an interest rate future option security with up-front payment of the premium."